Lib/test/test_asyncgen.py

```diff
@@ -1152,6 +1152,24 @@ async def run():
 
         self.loop.run_until_complete(run())
 
+    def test_async_gen_asyncio_anext_tuple_no_exceptions(self):
+        # StopAsyncIteration exceptions should be cleared.
+        # See: https://github.com/python/cpython/issues/128078.
+
+        async def foo():
+            if False:
+                yield (1, 2)
+
+        async def run():
+            it = foo().__aiter__()
+            with self.assertRaises(StopAsyncIteration):
+                await it.__anext__()
+            a, b = await anext(it, ('a', 'b'))
+            self.assertEqual(a, 'a')
+            self.assertEqual(b, 'b')
```

Comment:
```suggestion
            res = await anext(it, ('a', 'b'))
            self.assertEqual(res, ('a', 'b'))
```
As the default is tuple, lets compare it to tuple directly